torch/ao/quantization/backend_config/fbgemm.py

```diff
@@ -71,6 +72,49 @@
 )
 
 
+# ===================
+# | FBGEMM OP CONFIGS |
+# ===================
+
+
+def _get_fbgemm_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPatternConfig]:
+    configs = []
+    fbgemm_ops = [
+        torch.nn.InstanceNorm1d,
+        torch.nn.InstanceNorm2d,
+        torch.nn.InstanceNorm3d,
+        torch.nn.LayerNorm,
+    ]
+
+    for op in fbgemm_ops:
+        configs.append(
+            BackendPatternConfig(op)
+            .set_observation_type(ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT)  # noqa: E131
+            .set_dtype_configs(dtype_configs)
+        )
+
+    configs.append(
+        BackendPatternConfig(torch.nn.functional.layer_norm)
+        .set_observation_type(ObservationType.OUTPUT_USE_DIFFERENT_OBSERVER_AS_INPUT)  # noqa: E131
+        .set_dtype_configs(dtype_configs)
+        ._set_input_type_to_index({"weight": 2, "bias": 3})
+    )
+
+    configs.append(
+        BackendPatternConfig(torch.nn.functional.group_norm)
```

Comment:
I guess this changes functionality, since `group_norm` was previously not defined in `BackendConfig`? Maybe we should mention this in the PR description to be clear.